src/pages/people.js

```diff
@@ -1,8 +1,24 @@
-import { html, LitElement } from "lit";
+import { html, css, LitElement } from "lit";
 import { customElement } from "lit/decorators.js";
+import "/components/PersonItem.js";
 import { fetchPeople } from "/api.js";
 
 export class PagePeople extends LitElement {
+  static get styles() {
+    return css`
+      div {
+        display: grid;
+        grid-wrap: wrap;
+        gap: 15px;
+        justify-content: center;
+        grid-template-columns: 1fr 1fr 1fr;
+      }
```

Comment:
Para remover los estilos por defecto del `ul`

```suggestion
        grid-template-columns: 1fr 1fr 1fr;
        list-style: none;
        padding-left: 0;
        margin: 0;
      }
```